server/platform/services/searchengine/bleveengine/indexer/indexing_job_test.go

```diff
@@ -22,12 +23,15 @@ func TestBleveIndexer(t *testing.T) {
 	defer mockStore.AssertExpectations(t)
 
 	t.Run("Call GetOldestEntityCreationTime for the first indexing call", func(t *testing.T) {
+		logger := mlog.CreateConsoleTestLogger(t, true)
 		job := &model.Job{
 			Id:       model.NewId(),
 			CreateAt: model.GetMillis(),
 			Status:   model.JobStatusPending,
 			Type:     model.JobTypeBlevePostIndexing,
+			Logger:   mlog.CreateConsoleTestLogger(t, true),
 		}
+		job.InitLogger(logger)
```

Comment:
It appears a logger is added to the struct above, and also via InitLogger.  Is this intentional?

Reply:
Good catch, fixed in ba3ddd551b